### ISSUE TITLE
CSS improperly escaped

### ISSUE THREAD

**Issue by niccokunzmann** [niccokunzmann/open-web-calendar#396]
If you look at the example link, HTML escaping is active within the CSS.
This leads to problems as the `"` character is escaped:

https://openwebcal.liondor.org/calendar.html?css=.CALENDAR-INDEX-0%20%7B%20background-color%3A%20%23ffa348%3B%20%7D.CALENDAR-INDEX-1%20%7B%20background-color%3A%20%23f66151%3B%20%7D.CALENDAR-INDEX-2%20%7B%20background-color%3A%20%2357e389%3B%20%7D%20.CALENDAR-INDEX-3%20%7B%20background-color%3A%20%2362a0ea%3B%20%7D%0A.CALENDAR-INDEX-0%3A%3Abefore%20%7B%20content%3A%20%221%22%3B%20%7D%0A.CALENDAR-INDEX-1%3A%3Abefore%20%7B%20content%3A%20%222%22%3B%20%7D%0A.CALENDAR-INDEX-2%3A%3Abefore%20%7B%20content%3A%20%223%22%3B%20%7D%0A.CALENDAR-INDEX-3%3A%3Abefore%20%7B%20content%3A%20%224%22%3B%20%7D%0A&language=fr&tabs=month&title=Calendrier%20%C3%A9v%C3%A8nements%20public%20bassin&url=https%3A%2F%2Faccords.liondor.org%2Fremote.php%2Fdav%2Fpublic-calendars%2F2wCf7i4m8JLAddat%3Fexport&url=https%3A%2F%2Fcalendar.google.com%2Fcalendar%2Fical%2Fjkmv2ktbqct3m1m38alm2g1i98%2540group.calendar.google.com%2Fpublic%2Fbasic.ics&url=https%3A%2F%2Fcalendar.google.com%2Fcalendar%2Fical%2Flegalpon%2540gmail.com%2Fpublic%2Fbasic.ics&url=https%3A%2F%2Fframagenda.org%2Fremote.php%2Fdav%2Fpublic-calendars%2FAAjQqBYYDDLQpW3A%2F%3Fexport

This makes the CSS invalid:

![grafik](https://github.com/niccokunzmann/open-web-calendar/assets/564768/84f64190-fc22-4d74-bdc3-c2912a79ae75)

Basically valid CSS should be on the website, if the input is this valid CSS.

<small>
<hr/>
We're using <a href="https://polar.sh/niccokunzmann">Polar.sh</a> so you can upvote and help fund this issue. We receive the funding once the issue is completed & confirmed by you. Thank you in advance for helping prioritize & fund our work.
</small>

<a href="https://polar.sh/niccokunzmann/open-web-calendar/issues/396">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/396/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/396/pledge.svg">
</picture>
</a>
